packages/TelegramClient-Tests.package/TCTUsersTests.class/instance/testGetUserFor.st

```diff
@@ -0,0 +1,15 @@
+testing
+testGetUserFor
+
+	| user |
+	user := UndefinedObject.
```

Comment:
Why is user set to UndefinedObject initially? What would change, if this line didn't exist?

Reply:
the linter complains about undefined values. As `user` is defined asynchronously in the block below, it looks like it is never assigned to. The timeouts ensure that the user is properly assigned before accessing it.

Reply:
Alright, fair enough. Is `UndefinedObject` better than `nil`?

Reply:
we changed it to `nil`